### DISCUSSION TITLE
iOS 17 beta vai remover suporte para Web Apps instalados?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#2462] [Discussão]
Este artigo é sobre um iOS 17 beta 1 e aparentemente só afeta a União Europeia, onde ao tentar abrir um PWA instalado, mostra um alerta:
> Abrir 'WEB APP NAME' no Safari. 'WEB APP NAME' irá abrir em seu navegador padrão a partir de agora.

![image](https://github.com/frontendbr/forum/assets/736728/426ccc87-f779-49e8-bda0-6e5707713299)

O artigo também fala:
> Os sites instalados na tela inicial não conseguiram iniciar suas próprias atividades de nível superior, abrindo diretamente no navegador padrão, mesmo quando o navegador padrão é o navegador que o instalou.

https://open-web-advocacy.org/blog/apple-on-course-to-break-all-web-apps-in-eu-within-20-days/

-----

Mais um drama no suporte a PWAs no iOS, ou simplesmente a consolidação do anunciado? O receio geral é PWAs deixarem de existir no iOS completamente, pois a relevância deles já é muito questionada por todos.

**Comment by felquis** [comment]
Um tweet bem relevante no assunto https://twitter.com/mysk_co/status/1758196103470628983

![image](https://github.com/frontendbr/forum/assets/736728/7162582c-b784-4322-ba75-e46bd691e1d0)

**Comment by juliobitencourt** [comment]
Notícia que eu vi hoje

Apple says iOS 17.4 won’t remove Home Screen web apps in the EU after all

https://9to5mac.com/2024/03/01/apple-home-screen-web-apps-ios-17-eu/

**Comment by alianamartin879** [comment]
Deciphering legal nuances is similar to navigating the complexities of iOS 17 beta 1. Legal snags could appear in the same way that bugs impact PWAs in the EU. Think about the experience of https://assignmentshelp.uk/contract-law-assignment-help; they simplify complexity, just like beta testers do. Seek advice if legal alerts are echoed by technical issues. Delegating tasks to professionals guarantees a seamless experience, be it legal codes or troubleshooting technology.
